lib/includes_for_external_validation.rb

```diff
@@ -0,0 +1,76 @@
+# gems
+require 'aasm'
+require 'devise'
+require 'devise/orm/mongoid'
+require 'carrierwave'
+require 'carrierwave/mongoid'
+require 'health-data-standards'
```

Comment:
Since HDS + QME are in the Gemfile here + full dependencies of our app, can we remove these requires?